runtime/compiler/env/j9method.cpp

```diff
@@ -2366,12 +2387,14 @@ TR_ResolvedJ9Method::TR_ResolvedJ9Method(TR_OpaqueMethodBlock * aMethod, TR_Fron
    construct();
    }
 
+#if defined(JITSERVER_SUPPORT)
```

Comment:
Let's eliminate the disparaging comment "(because the way constructors work in C++ is awful)"